### PR TITLE
Add doctest, missing import, refs, and stream -> io for multimedia

### PR DESCRIPTION
**Pull request by kshyatt**
the `base64` encoding stuff for `stringmime` needs the import

### CODE REVIEW

File: base/multimedia.jl [hunk 1]
```diff
@@ -15,7 +15,7 @@ export AbstractDisplay, display, pushdisplay, popdisplay, displayable, redisplay
 # struct MIME{mime} end
 # macro MIME_str(s)
 import Base: MIME, @MIME_str
-
+import Base64
```
Comment (fredrikekre) [inline base/multimedia.jl]: Isn't this a bit weird? I thought we moved `Base64` out because nothing in `Base` depend on it?

Comment (KristofferC) [reply base/multimedia.jl]: I think stuff in Base depend on it (that's why it is in a bit weird place in sysimg.jl).

Comment (fredrikekre) [reply base/multimedia.jl]: Why is it needed here? For the doctests? Or is there a method in this file that the test don't cover?

**Comment by kshyatt** [comment]
There's a method (stringmime for binary mime types) which isn't covered by
the tests.

**Comment by kshyatt** [comment]
Is this ok to go? Did I answer your question, @fredrikekre ?

**Pull request merged**